src/overlays/actors/ovl_Obj_Switch/z_obj_switch.h

```diff
@@ -17,21 +17,21 @@ typedef enum {
 } ObjSwitchType;
 
 typedef enum {
-    /* 0 */ OBJSWITCH_SUBTYPE_FLOOR_0,
-    /* 1 */ OBJSWITCH_SUBTYPE_FLOOR_1,
-    /* 2 */ OBJSWITCH_SUBTYPE_FLOOR_2,
-    /* 3 */ OBJSWITCH_SUBTYPE_FLOOR_3
+    /* 0 */ OBJSWITCH_SUBTYPE_FLOOR_NORMAL,     // Normal Yellow Switch
+    /* 1 */ OBJSWITCH_SUBTYPE_FLOOR_TOGGLE,     // On/Off Red Switch
+    /* 2 */ OBJSWITCH_SUBTYPE_FLOOR_RESET_OFF,  // Reset when not stood on, blue switch (default state: off)
+    /* 3 */ OBJSWITCH_SUBTYPE_FLOOR_RESET_ON    // Reset when not stood on, blue switch (default state: on)
```

Comment:
does the default state being off/on mean "is the switch up or down", or "the switch flag being cleared or set"?

Reply:
It means "is the switch up or down", but if you use "On" by default it'll go back to off if there's nothing on the switch

One use case I see is if you want to have a floor switch that resets and it has a small crate on it by default to avoid a cutscene when you load the room